templates/react/entrypoints/content.ts

```diff
@@ -0,0 +1,7 @@
+export default defineContentScript({
+  matches: ['*://*.google.com/*'],
+  runAt: 'document_start',
```

Comment:
Let's exclude `runAt` and just use the default. Otherwise everything else looks good.

Reply:
Done.